lib/plugins/aws/package/compile/events/httpApi/index.test.js

```diff
@@ -107,6 +111,25 @@ describe('HttpApiEvents', () => {
     });
   });
 
+  describe('Detailed Metrics', () => {
+    let cfHttpApiStage;
+
+    before(() =>
+      runServerless({
```

Comment:
Let's reuse one of the previous `runServerless` runs for testing that (those runs are expensive, therefore idea is to configure new run only if there's no other way to effectively test the feature)

Reply:
Done, I put my tests in the runServerless used for the logs